### ISSUE TITLE
Learned shaplets do not match the original time serries scale  

### ISSUE THREAD

**Issue by hbout** [rtavenar/keras_shapelets#1]
I am wondering why the learned shaplets do not match the original time series scale. This is  clear once the shaplets are plotted on the original time series.

![sphx_glr_plot_shapelet_locations_001](https://user-images.githubusercontent.com/28430946/56510743-a8fda780-6522-11e9-82e5-56ce5374c065.png)

**Comment by rtavenar** [comment]
Hi @hbout 

Well, there is nothing in the loss function that force the shapelets to match the scale of the time series, so this can definitely happen.
